src/ZipkinOpenTracing/Tracer.php

```diff
@@ -41,6 +38,59 @@ public function __construct(ZipkinTracing $tracing)
     {
         $this->tracer = $tracing->getTracer();
         $this->propagation = $tracing->getPropagation();
+        $this->scopeManager = new ScopeManager();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getScopeManager()
+    {
+        return $this->scopeManager;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getActiveSpan()
+    {
+        $activeScope = $this->scopeManager->getActive();
+        if ($activeScope === null) {
+            return null;
+        }
+
+        return $activeScope->getSpan();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function startActiveSpan($operationName, $options = [])
```

Comment:
We need a test for this method. Can you please add it @eligijusvitkauskas-home24?